PsfShimMonitor/README.md

```diff
@@ -0,0 +1,30 @@
+# PSF Monitor
+This sub-project provides for a Graphical User Interface tool, in the style of ProcessMonitor, that integrates events output by TraceFixup with those of kerenel level tracing.
+
+Used in conjuction with an application launched by PsfLauncher and the TraceFixup dll configured to output events, this makes it easier to trace the application calls that may ultimately require fixups.
+
+## Documentation
+See the readme.md for PsfLauncher for documentation on how to use PsfLauncher in conjuction with TraceFixup to produce tracing events in your package.
+
+The PsfLauncher documentation shows how to integrate the monitor inside your package and have it automatically run when you start the application to be traced.
+You may, however, run PSF Monitor externally from your package as long as the package is configured with the TraceFixup shim.
+
+
+## [License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE)
+Code licensed under the [MIT License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE).
+
+## Contribute

```

Comment:
Please remove the license and the contribute sections, it is a dup to the one we have for the whole repo.